x-pack/legacy/plugins/reporting/server/config/index.ts

```diff
@@ -204,11 +78,12 @@ export const buildConfig = (
   };
 
   // spreading arguments as an array allows the return type to be known by the compiler
```

Comment:
Small nit: comment no longer seems valid...

Reply:
I think it's just confusing the way Github renders the diff, but the comment is still valid to lines 82 + 84